vividus-plugin-lighthouse/src/main/java/org/vividus/lighthouse/model/PerformanceMetricRule.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.lighthouse.model;
+
+import java.math.BigDecimal;
+
+import org.jbehave.core.annotations.AsParameters;
+import org.vividus.steps.ComparisonRule;
+
+@AsParameters
```

Comment:
just curious: is it required?

Reply:
Caused by: org.jbehave.core.steps.ParameterConverters$ParameterConversionFailed: No parameter converter for java.util.List<org.vividus.lighthouse.model.PerformanceMetricRule>
        at org.jbehave.core.steps.ParameterConverters.convert(ParameterConverters.java:411) ~[jbehave-core-5.1.1-alpha.2.jar:?]
        at org.vividus.steps.ParameterConvertersDecorator.convertAdaptedValue(ParameterConvertersDecorator.java:81) ~[vividus-engine-0.5.11-SNAPSHOT.jar:0.5.11-SNAPSHOT]
        at org.vividus.steps.ParameterConvertersDecorator.convert(ParameterConvertersDecorator.java:54) ~[vividus-engine-0.5.11-SNAPSHOT.jar:0.5.11-SNAPSHOT]
        at org.jbehave.core.steps.StepCreator$ParametrisedStep.convertParameterValues(StepCreator.java:970) ~[jbehave-core-5.1.1-alpha.2.jar:?]
        at org.jbehave.core.steps.StepCreator$ParametrisedStep.parametriseStep(StepCreator.java:954) ~[jbehave-core-5.1.1-alpha.2.jar:?]
        at org.jbehave.core.steps.StepCreator$ParametrisedStep.perform(StepCreator.java:891) ~[jbehave-core-5.1.1-alpha.2.jar:?]

as I understand its used to determine whether to perform field mapping or to directly call converter

Reply:
we can think of defaulting to field mapping if parameter converter is not available

Reply:
https://jbehave.atlassian.net/browse/JBEHAVE-1581